### ISSUE TITLE
struct_array_notation misreading top-level scopes

### ISSUE THREAD

**Issue by lmajano** [cflint/CFLint#542]
If you have code like this

```java
variables.CFLINT_VERSION = "1.3.0";
```

It will mis-report it that I need to quote it in the struct.  This should be relaxed in my opinion for top level scopes or storage scopes. Thoughts?

**Comment by TheRealAgentK** [comment]
Is that because you’re seeing a notation

variables.CFLINT_VERSION

as something similar to a string constant in Java?

**Comment by lmajano** [comment]
That was my example but it is for any case.

Luis Majano
CEO
Ortus Solutions, Corp
www.ortussolutions.com
P/F: 1-888-557-8057
________________________________

**Comment by ryaneberly** [comment]
@lmajano , I agree.  This rule was not intended for scopes.    Fixed in dev branch.
